<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDynamicModellerModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerDynamicModellerModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="ParameterNodeLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Parameter node:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="ParameterNodeComboBox">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLDynamicModellerNode</string>
        </stringlist>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rule:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="RuleComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="InputNodesCollapsibleButton">
     <property name="text">
      <string>Input nodes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ParametersCollapsibleButton">
     <property name="text">
      <string>Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="OutputNodesCollapsibleButton">
     <property name="text">
      <string>Output nodes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCheckablePushButton" name="ApplyButton">
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="checkState">
      <enum>Qt::Unchecked</enum>
     </property>
     <property name="checkBoxControlsButtonToggleState">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCheckablePushButton</class>
   <extends>ctkPushButton</extends>
   <header>ctkCheckablePushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPushButton</class>
   <extends>QPushButton</extends>
   <header>ctkPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerDynamicModellerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ParameterNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
